components/scream/src/physics/p3/tests/p3_main_unit_tests.cpp

```diff
@@ -56,11 +56,11 @@ static void run_bfb_p3_main_part1()
   constexpr Scalar sup_lower = -0.1;
 
   P3MainPart1Data isds_fortran[] = {
-    //            kts, kte, ktop, kbot, kdir, do_predict_nc,        dt
-    P3MainPart1Data(1,  72,    1,   72,    1, false,         1.800E+03),
-    P3MainPart1Data(1,  72,    1,   72,    1, true,          1.800E+03),
-    P3MainPart1Data(1,  72,   72,    1,   -1, false,         1.800E+03),
-    P3MainPart1Data(1,  72,   72,    1,   -1, true,          1.800E+03),
+    //            kts, kte, ktop, kbot, kdir, do_predict_nc, do_prescribed_CCN,       dt
+    P3MainPart1Data(1,  72,    1,   72,    1, false,          true,           1.800E+03),
+    P3MainPart1Data(1,  72,    1,   72,    1, true,           true,           1.800E+03),
+    P3MainPart1Data(1,  72,   72,    1,   -1, false,          false,          1.800E+03),
+    P3MainPart1Data(1,  72,   72,    1,   -1, true,           false,          1.800E+03),
   };
```

Comment:
@hassanbeydoun - I think that do_predict_nc should only be true when do_prescribed_CCN is false and vice versa. Do the columns for do_predict_nc and do_prescribed_CCN just above and on the right hand panel make sense to you?

Reply:
@PeterCaldwell, yes it makes sense since the code never forces these two logicals be of opposite value.